### DISCUSSION TITLE
How to export command as a utf-8 csv

### DISCUSSION THREAD

**Discussion by kam-stand** [microsoft/go-sqlcmd#530] [Q&A]
I need to run a query and then export a csv utf-8 format. However, when I tried to follow this generic command:
`Sqlcmd -S "10.00.00.00" -d "EmployeeDatabase" -U "User1" -P "Password" -i "Script1.sql" -W -f 65001 -h -1 -s "," -o "c:\File.csv"
`
in the latest version of sqlcmd it does not have the lower case "f" flag.

any advice is much appreciated

**Comment by shueybubbles** [comment]
I believe the output is UTF8 by default, since Go's native string format is UTF 8

**Reply by kam-stand** [reply]
right but the output comes out into one singular column per row 
![image](https://github.com/microsoft/go-sqlcmd/assets/123669329/164c53e4-f670-48d4-b20b-ea1459b32333)
As you can see from this image the data is all in the "A" column and inside each cell in the "A" column it is put into a format that describes the columns

**Comment by shueybubbles** [comment]
if the file has commas in it correctly, I'm not sure what sqlcmd could be doing wrong.
